lib/Echidna/Transaction.hs

```diff
@@ -115,7 +115,7 @@ liftSH = S.state . runState . zoom hasLens
 -- 'Transaction's \"on-chain\".
 setupTx :: (MonadState x m, Has VM x) => Tx -> m ()
 setupTx (Tx c s r v) = S.state . runState . zoom hasLens . sequence_ $
-  [ result .= Nothing, state . pc .= 0, state . stack .= mempty, state . gas .= 0xffffffff
+  [ result .= Nothing, state . pc .= 0, state . stack .= mempty, state . memory .= mempty, state . gas .= 0xffffffff
```

Comment:
is this supposed to be in this PR? also the associated test

Reply:
Hmm, not sure how this got in, yeah, it's a fix I was testing but not relevant here

Reply:
Done